apps/client/src/pages/CheckerPage.js

```diff
@@ -57,6 +58,29 @@ const CheckerPage = ({ checker, matomoTrackEvent, resetChecker, topic }) => {
       }
     }
 
+    // Make the app backwards compatible:
+
+    // This section does not exist anymore in the IMTR flow
+    const isOldSectionActive = hasIMTR && isActive(sections.LOCATION_RESULT);
+    const newFinishedComponents = finishedComponents.filter(
+      (section) => section !== sections.LOCATION_INPUT
+    );
+
+    if (isOldSectionActive) {
+      console.warn(
+        "Resetting components, because an old section was found active"
+      );
```

Comment:
Can we not filter out all activeComponents that no longer exists? This implementation only takes care of 1 if I'm reading this correctly.
Something in the lines of `if (!currentlyKnownComponents[activeComponent])...`.

Reply:
I was thinking about this, but decided to drop this since we're going to implement this in a very different way with the new state refactor next sprint.